redesign/js/main.js

```diff
@@ -13,6 +13,23 @@ require( './view_helpers' );
 require( './scrolling_effects' );
 require( './background' );
 
+var trackingSwitch = $( '#tracking-switch' );
+if( tracking.shouldTrack() ) {
+	// TODO germanify
+	trackingSwitch.text( 'Dont\'t track me!' );
```

Comment:
Instead of the hardcoded texts you can use `Messages.t( 'do-not-track' )` and then add an empty entry to the `i18n.json` file so we know where to add missing translations!